### PR TITLE
Create wade.md

### PR DESCRIPTION
**Pull request by wadechandler**
This is a bomb file, and so creative!

@tbrantb @rmendu

### CODE REVIEW

**Comment by githubteacher** [comment]
Hi @wadechandler I see your file! Any restaurant recommendations?

**Comment by mock** [comment]
how big is your file already?

File: wade.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Hello #
+
+Do you see my file?
```
Comment (wadechandler) [inline wade.md:3]: This line is so cool

File: wade.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Hello #
+
```
Comment (wadechandler) [inline wade.md:2]: Here is a line with nothing on it...

**Review by tbrantb — APPROVED** [review]
ok

**Review by leachryanb — APPROVED** [review]
You should be banned from the internet!

**Comment by wadechandler** [comment]
Forgot to close

**Pull request closed without merging**